cookbook/catalog/common/remove.rst

```diff
@@ -0,0 +1,38 @@
+How to Remove Non-Product Objects
+=================================
+
+To remove one or many objects, we provide a dedicated service which provides methods 'remove' and 'removeAll' through the implementation of ``Akeneo\Component\StorageUtils\Remover\RemoverInterface`` and ``Akeneo\Component\StorageUtils\Remover\BulkRemoverInterface``.
+
+Use the Remover to Remove a Single Object
```

Comment:
http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html ?